.gitignore

```diff
@@ -1,7 +1,3 @@
 /node_modules/
 /typings/
-/jspm_packages/
-/dist/
-npm-debug.log
-.sw*
-.idea
```

Comment:
Thing is, when we get more contributors, they will likely not have it stored in a global gitignore and will likely add these files.